asyncapi/github/dispatcher_service.bal

```diff
@@ -43,36 +39,24 @@ service class DispatcherService {
         _ = self.services.remove(serviceType);
     }
 
-    // We are not using the (@http:payload GenericEventWrapperEvent g) notation because of a bug in Ballerina.
-    // Issue: https://github.com/ballerina-platform/ballerina-lang/issues/32859
-    resource function post .(http:Caller caller, http:Request request) returns http:Response|error? {
-        json payload = check <@untainted> request.getJsonPayload();
-        byte [] binaryPay = <@untainted>  payload.toString().toBytes();
-        string eventName = check  request.getHeader("X-GitHub-Event");
-        if (request.hasHeader("X-Hub-Signature-256")) {
-            string secret = check  request.getHeader("X-Hub-Signature-256");
-            string trimmedSecret = secret.substring(7, secret.length());
-            byte [] output = check crypto:hmacSha256(binaryPay, self.listenerConfig.webhookSecret.toBytes());
-            string computedDigest = output.toBase16();
-            if (trimmedSecret.length() != computedDigest.length() || trimmedSecret !== computedDigest) {
-                // Validate secret with X-Hub-Signature-256 header for intent verification
-                log:printError("Signature verification failure");
-                http:Response response = new;
-                response.statusCode = http:STATUS_UNAUTHORIZED;
-                response.setPayload("Signature verification failure");
-                return response;
-            }
+    remote function onEventNotification(websub:ContentDistributionMessage event)
+    returns Acknowledgement|websub:SubscriptionDeletedError|error? {
+        json payload = check event.content.ensureType(json);
+        map<string|string[]>? headers = event.headers;
+        string eventName = "";
+        if headers is map<string|string[]> {
+            string|string[] events = headers.get("X-Github-Event");
+            eventName = events is string ? events : events[0];
         }
         GenericDataType|error genericDataType = payload.cloneWithType(GenericDataType);
         if (genericDataType is error) {
             log:printError("Unsupported Event Type : " + eventName);
             return genericDataType;
         } else {
             check self.matchRemoteFunc(genericDataType, eventName);
-            http:Response response = new;
-            response.statusCode = http:STATUS_OK;
-            response.setPayload("Event acknoledged successfully");
-            return response;
+            return {
+                body: {"message": "Event acknoledged successfully"}
```

Comment:
spelling mistake here.
Suggestion: Event acknowledged

Reply:
Fixed in https://github.com/ballerina-platform/asyncapi-triggers/commit/e0c56638e7d25694ff986c19c894b01fc776abb8